jax/_src/scipy/stats/_core.py

```diff
@@ -46,6 +46,40 @@ def mode(a: ArrayLike, axis: int | None = 0, nan_policy: str = "propagate", keep
   Returns:
     A tuple of arrays, ``(mode, count)``. ``mode`` is the array of modal values,
     and ``count`` is the number of times each value appears in the input array.
+
+  Examples:
+    >>> x = jnp.array([2, 4, 1, 1, 3, 4, 4, 2, 3])
+    >>> mode, count = jax.scipy.stats.mode(x)
+    >>> mode, count
+    (Array(4, dtype=int32), Array(3, dtype=int32))
+
+    For multi dimensional arrays, ``jax.scipy.stats.mode`` compuptes the ``mode``
+    and the corresponding ``count`` along ``axis=0``:
+
+    >>> x1 = jnp.array([[1, 2, 1],
+    ...                 [3, 1, 3],
+    ...                 [1, 2, 2]])
```

Comment:
The axis behavior would be clearer if you used a non-square array here.

Reply:
Thanks! Updated the examples with non-square matrix.